src/pieces/commands/assets.py

```diff
@@ -51,7 +51,7 @@ def list_assets(max_assets:int=10):
     asset_list = get_assets_info_list()
     
     for i, name in enumerate(asset_list, start=1):
```

Comment:
I think the name property can be null, also calling the asset 'name' as a variable is a bit confusing so can we adjust this to:

```python
for i, asset in enumerate(asset_list, start=1):
  print(f"{i}: {asset.get('name', 'New Asset')}")
```

Reply:
fix it in the other pull request to keep things clean I added this thank you for your review

https://github.com/pieces-app/cli-agent/pull/77/commits/8aea0b4740e64450f8524753b77b7800356e8eaf